salsa_dancing_molecules/energy.py

```diff
@@ -0,0 +1,25 @@
+"""Functionfile for energy.
+
+Functions to get total energy, potential energy and
+kinetic energy from an atoms object.
+"""
+
+
+def get_energy(atoms):
```

Comment:
This function return the energy per atom if I've understood it correctly. Should this be reflected in the function name or its documentation comment? The same goes for the other functions.